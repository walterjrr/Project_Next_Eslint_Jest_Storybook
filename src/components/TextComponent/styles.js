/* eslint-disable no-unused-vars */
import styled, { css } from 'styled-components';

export const Container = styled.p`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
  `};
`;
