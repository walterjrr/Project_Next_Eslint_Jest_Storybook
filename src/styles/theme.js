export const theme = {
  colors: {
    primaryColor: '#0A1128',
    secondaryColor: '#dc143c',
    white: '#ffffff',
  },
  fonts: {
    family: {
      default: "'Opens Sans', sans-serif",
      secondary: "'Montserrat', sans-serif",
    },
  },
  spacings: {},
};
