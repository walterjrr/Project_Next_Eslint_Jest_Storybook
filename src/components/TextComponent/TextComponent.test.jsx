import { renderTheme } from '../../styles/render-theme';
import { TextComponent } from '.';
import { screen } from '@testing-library/react';

describe('<TextComponent />', () => {
  it('should render a text', () => {
    renderTheme(<TextComponent>Children</TextComponent>);
    expect(screen.getByText('Children')).toBeInTheDocument();
  });

  it('should match a snapshot', () => {
    const { container } = renderTheme(<TextComponent>Children</TextComponent>);
    expect(container).toMatchSnapshot();
  });
});
