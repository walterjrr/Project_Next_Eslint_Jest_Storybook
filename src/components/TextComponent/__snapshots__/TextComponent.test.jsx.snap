// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextComponent /> should match a snapshot 1`] = `
.c0 {
  font-size: 2.4rem;
}

<div>
  <p
    class="c0"
  >
    Children
     
  </p>
</div>
`;

exports[`<TextComponent /> should render a text 1`] = `
.c0 {
  font-size: 2.4rem;
}

<div>
  <p
    class="c0"
  >
    Children
     
  </p>
</div>
`;
